doc/developers/develop.rst

```diff
@@ -618,15 +620,28 @@ X_types (default=['2darray'])
     ``'categorical'`` data. For now, the test for sparse data do not make use
     of the ``'sparse'`` tag.
 
+It is unlikely that the default values for each tag will suit the needs of
+your specific estimator. There are two ways to override the defaults in your
+own estimator:
 
-To override the tags of a child class, one must define the `_more_tags()`
-method and return a dict with the desired tags, e.g::
+* If your estimator inherits from :class:`~sklearn.base.BaseEstimator`, which
+  is recommended, you can define a `_more_tags()` method which returns a dict
+  with the desired overridden tags. For example::
 
     class MyMultiOutputEstimator(BaseEstimator):
 
         def _more_tags(self):
             return {'multioutput_only': True,
                     'non_deterministic': True}
+  
+  Any tag that is not in `_more_tags()` will just default to the value
+  documented above.
+
+* If your estimator does not inherit from :class:`~sklearn.base.BaseEstimator`,
+  you will need to implement a `_get_tags()` method which returns a dict,
+  similar to `_more_tags()`. Note however that **all tags must be present in
```

Comment:
I find that "similar to `_more_tags()`" to be confusing. I would rather say that it should return a dict with all the necessary tags for that estimator, including the default tags typically defined in `BaseEstimator` and other scikit-learn mixin classes.